src/main/kotlin/com/jetbrains/snakecharm/inspections/SmkRedundantCommaInspection.kt

```diff
@@ -4,49 +4,62 @@ import com.intellij.codeInspection.*
 import com.intellij.openapi.command.WriteCommandAction
 import com.intellij.openapi.project.Project
 import com.intellij.psi.PsiElement
-import com.intellij.psi.PsiWhiteSpace
 import com.intellij.psi.TokenType
 import com.intellij.psi.util.elementType
-import com.intellij.psi.util.parents
-import com.intellij.psi.util.siblings
 import com.jetbrains.python.PyTokenTypes
 import com.jetbrains.python.psi.PyArgumentList
 import com.jetbrains.snakecharm.SnakemakeBundle
 import com.jetbrains.snakecharm.lang.psi.SmkArgsSection
-import com.jetbrains.snakecharm.lang.psi.SmkRuleOrCheckpointArgsSection
 
-class SmkRedundantComaInspection : SnakemakeInspection() {
+class SmkRedundantCommaInspection : SnakemakeInspection() {
     override fun buildVisitor(
         holder: ProblemsHolder,
         isOnTheFly: Boolean,
         session: LocalInspectionToolSession
     ) = object : SnakemakeInspectionVisitor(holder, session) {
 
-        private fun problemRegiser(psiEl: PsiElement) = holder.registerProblem(
-                psiEl,
-                SnakemakeBundle.message("INSP.NAME.redundant.coma.title"),
-                ProblemHighlightType.LIKE_UNUSED_SYMBOL,
-                fix
+        private fun problemRegiser(psiEl: PsiElement?) = psiEl?.let {
+            holder.registerProblem(
```

Comment:
Not a good style - create method with nullable element which does nothing. You have the only one usage of this method. Better to make it's parameter not null here.

Reply:
Done